frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -119,81 +68,18 @@ const CourseDescription = ({ structure }) => {
     );
   }
 
-  if (id === "explore") return <div>This is the explore page</div>;
-  if (id === "search") return <SearchCourse />;
-  const addToPlanner = (type) => {
-    const data = {
-      courseCode: course.code,
-      courseData: {
-        title: course.title,
-        type,
-        termsOffered: course.terms,
-        UOC: course.UOC,
-        plannedFor: null,
-        prereqs: course.raw_requirements,
-        isLegacy: course.is_legacy,
-        isUnlocked: true,
-        warnings: [],
-        handbookNote: course.handbook_note,
-        isAccurate: course.is_accurate,
-      },
-    };
-    dispatch(addToUnplanned(data));
-    setLoading(true);
-    setTimeout(() => {
-      setLoading(false);
-    }, 1000);
-  };
-
-  const removeFromPlanner = () => {
-    dispatch(removeCourse(id));
-    setLoading(true);
-    setTimeout(() => {
-      setLoading(false);
-    }, 1000);
-  };
-
   return (
     <div className="cs-description-root">
       {!pageLoaded ? (
-        <Loading />
+        <LoadingSkeleton />
       ) : (
         <>
           <div className="cs-description-content">
             <div className="cs-desc-title-bar">
               <Title level={2} className="text">
                 {id} - {course.title}
               </Title>
-              {courseInPlanner ? (
-                <Button
-                  type="secondary"
-                  loading={loading}
-                  onClick={removeFromPlanner}
-                  icon={<StopOutlined />}
-                >
-                  Remove from planner
-                </Button>
-              ) : (
-                <Dropdown.Button
-                  overlay={() => (
-                    <PlannerDropdown
-                      courseCode={course.code}
-                      structure={structure}
-                      addToPlanner={addToPlanner}
-                    />
-                  )}
-                  loading={loading}
-                  onClick={() => {
-                    addToPlanner("Uncategorised");
```

Comment:
where did this logic go?

Reply:
Removed the dropdown for now since we aren't considering the course's type (i.e. Core Course, Gen Ed) for the term planner atm. Will revisit this down the road